CONTRIBUTING.md

```diff
@@ -191,7 +192,7 @@ When making a contribution, adhering to the [GitHub flow](https://guides.github.
    ```bash
    git pull --rebase upstream <dev-branch>
    ```
-
+> **Note:** When working on the frontend. Activist recommends running `yarn nuxi typecheck` from within the `frontend` directory to confirm your changes are type-safe. Existing TS errors may be ignored. PR's to fix these are always welcome.
```

Comment:
This reads really well, @TeddyGavi! One thing is that `activist` needs a lower case `a` 🙃